src/py/flwr/superexec/exec_servicer.py

```diff
@@ -138,15 +138,13 @@ def StopRun(
 
         # Exit if `run_id` not found
         if not state.get_run(request.run_id):
-            context.abort(
-                grpc.StatusCode.NOT_FOUND, f"Run ID {request.run_id} not found"
-            )
+            return StopRunResponse(success=False, reason="Run not found")
```

Comment:
We use `context.abort` in the other RPC methods like `StreamLogs` (on [this line](https://github.com/adap/flower/blob/0ea7a0e0ae44219426c4e056a57c5328ecbd37de/src/py/flwr/superexec/exec_servicer.py#L95)). Should we use the same `context.abort` for consistency?